src/sentry/static/sentry/app/views/settings/organization/organizationSettingsLayout.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import {RouteComponentProps} from 'react-router/lib/Router';
+
+import OrganizationSettingsNavigation from 'app/views/settings/organization/organizationSettingsNavigation';
+import SettingsLayout from 'app/views/settings/components/settingsLayout';
+
+type Props = RouteComponentProps<{orgId: string}, {}> & {
+  children: React.ReactNode;
+};
+
+function OrganizationSettingsLayout(props: Props) {
+  return (
+    <SettingsLayout
+      {...props}
+      renderNavigation={() => <OrganizationSettingsNavigation {...props} />}
```

Comment:
Please, is it necessary to forward the children here?

Reply:
I think not, I simplified it a bit 👍